src/manifest/manifest.go

```diff
@@ -177,8 +177,17 @@ func (m *Manifest) Save() error {
 	return nil
 }
 
-func (m *Manifest) IsBranchAllowed(branch *model.Branch) bool {
-	return m.Content.AllowedBranches.IsBranchAllowed(branch)
+func (m *Manifest) IsObjectIgnored(object model.Object) bool {
```

Comment:
Namiesto `IsBranchAllowed` som to zovseobecnil na `IsObjectIgnored`.
Je mozne ignorovat `branch`, alebo `componentId`.